.github/workflows/R.yml

```diff
@@ -30,10 +30,78 @@ env:
   NODE_AUTH_TOKEN: ${{secrets.NODE_AUTH_TOKEN}}
 
 jobs:
+
+  r-test-extension:
```

Comment:
This seems to duplicate extension building logic here only for the testing of the R extension, would it perhaps make more sense to create a new workflow (`Extensions.yml`) and move all extension installation/testing code to there?